scene/realistic-light-and-shadows/gradle/wrapper/gradle-wrapper.properties

```
@@ -0,0 +1,5 @@
+distributionBase=GRADLE_USER_HOME
+distributionPath=wrapper/dists
+distributionUrl=https\://services.gradle.org/distributions/gradle-5.0-bin.zip
```

Another spot we should update the gradle version:

```suggestion
distributionUrl=https\://services.gradle.org/distributions/gradle-6.5.1-bin.zip
```